libs/csm/rest/csm_rest_quota.py

```diff
@@ -51,7 +55,7 @@ def get_user_quota(self, uid, quota_type, **kwargs):
         :return: response
         """
         self.log.info("Get IAM user request....")
-        if "headers" in kwargs.keys():
+        if "headers" in kwargs.items():
```

Comment:
This will give incorrect results. You can revert back this change or u can use as below
key, value = kwargs.item()
"headers" in k

Reply:
reverted back